src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
Well, okay, I guess.
(The purist in me decries storing derived files... heck, I was debating trying to find a way to generate directly from the .nlp and .resx files).

I already know how to drop it in the source directory, do you want it to appear in the listing in VS too (not too sure about how to do that, yet)?